src/sortStudents.ts

```diff
@@ -1,16 +1,58 @@
+// import { sortAndDeduplicateDiagnostics } from "typescript";
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function findAverage(arr: number[]): number {
+  return arr.reduce((acc, curr) => acc + curr, 0) / arr.length;
```

Comment:
you can save 1 iteration here) if you do not set default value then it starts from the 1st element in array right away
```suggestion
  return arr.reduce((acc, curr) => acc + curr) / arr.length;
```